gravitee-am-ui/src/app/domain/settings/users/user/profile/profile.component.ts

```diff
@@ -321,4 +321,19 @@ export class UserProfileComponent implements OnInit {
     user.accountLockedAt = null;
     user.accountLockedUntil = null;
   }
+
+  setForceResetPassword(e: any): void {
+    this.user.forceResetPassword = e.checked;
+    this.formChanged = true;
+  }
+
+  passwordReset(): boolean {
+    //Sometimes time differ by 1 millisecond
```

Comment:
I don't understand this comment. Could you explain?

Reply:
I had to "cut" milliseconds because during creation of object, the createdAt and lastPasswordUpdate differs by 1 milliseconds and when I compared them It is false.

Reply:
Ah, I think this comment can be removed because it is clearly visible in the code below.